app/js/main.js

```diff
@@ -19,13 +47,13 @@ app.controller('courseGenerator', ['$scope', '$http', '$routeParams', 'courseGen
 			console.log($scope.courseGeneratorData.course);
 			//LOOPING THROUGH THE RETURNED DATA AND PUSHING EACH OBJECT INDIVIDUALLY INTO theCourses ARRAY
 		});
-	
-		$scope.addRubric = function(course){
-			location.assign('/addRubric', course);
-			console.log('Course Data ', course);
-		}
 
 
+	$scope.addRubric = function(course){
+		//location.assign('/addRubric', course);
+		window.location.hash = "/addRubric";
```

Comment:
Angular provides `$location.path('/addRubric')` that should be used instead of `window.location.hash = "/addRubric";`. window.location should only be used to navigate the user away from the angular SPA.